src/compas_fab/robots/semantics.py

```diff
@@ -41,6 +43,37 @@ def __init__(
         self.disabled_collisions = disabled_collisions or set()
         self.group_states = group_states or {}
 
+    @property
+    def data(self):
+        data = {
+            "robot_model": self.robot_model,
+            "groups": self.groups,
+            "main_group_name": self.main_group_name,
+            "passive_joints": self.passive_joints,
+            "end_effectors": self.end_effectors,
+            "disabled_collisions": sorted(self.disabled_collisions),
```

Comment:
Why are these sorted? not that is bad, just curious

Reply:
Okay, I removed the underscore in the keys of the serialized dictionary. But kept the original variable names with underscore.

Skipped `current_ik ` serialization.

The sorted thing is to improve the diff-ability of the text output. Otherwise random ordering on save can create phantom changes that show up in diff, but in fact there are no real changes to the data.